components/table/TableRow.js

```diff
@@ -53,6 +54,10 @@ const factory = (Checkbox) => {
       if (onChange) {
         return this.renderInput(key, value);
       } else if (value) {
+        if (this.props.model[key].render) {
+          const disabled = true;
```

Comment:
It needs value as well but sure, then the parameter order is irrelevant.  You're right that the variable assignment was purely to make the code self-documenting.
```
this.props.model[key].render({ 
  index: this.props.index,
  disabled: true,
  key,
  value,
});
```
To be honest, I want the change in first so that I could use your playground to write an example that I know works and then I could push that as the new playground example code.